test/.env.test

```diff
@@ -39,6 +39,10 @@ GOOGLE_MAP_API_KEY=test_google_map_api_key
 
 HERE_API_KEY=test_here_api_key
 
+IMGUR_CLIENT_ID=test_imgur_client_id
+IMGUR_CLIENT_SECRET=test_imgur_client_secret
+
+LASTFM_KEY=test_lastfm_key
```

Comment:
It looks like a duplicate LASTFM_KEY from resolving the merge conflict.  You can remove "LASTFM_KEY=test_lastfm_key"